// astrology-chart-data-generator/src/_tests_/test_examples/brazil.js
export const testExampleBrazil = {
  location: "Brazil",
  data: {
    longitude: -34.900002,
    latitude: -8.05,
    dateTime: {
      year: 1987,
      month: 5,
      day: 23,
      hour: 11,
      minute: 30,
      offset: 3,
    },
  },
  angles: {
    ascendant: {
      zodiacDegree: `29° 30' 22" Leo`,
    },
    descendant: {
      zodiacDegree: `29° 30' 22" Aquarius`,
    },
    midheaven: {
      zodiacDegree: `5º 13' 12\" Gemini`,
    },
    lowerMidheaven: {
      zodiacDegree: `5º 13' 12\" Sagittarius`,
    },
  },
  planets: {
    sun: {
      zodiacDegree: `1° 56' 13" Gemini`,
    },
    moon: {
      zodiacDegree: `14° 51' 57" Aries`,
    },
    mercury: {
      zodiacDegree: `19° 27' 50" Gemini`,
    },
    venus: {
      zodiacDegree: `7° 24' 2" Taurus`,
    },
    mars: {
      zodiacDegree: `1° 36' 45" Cancer`,
    },
    jupiter: {
      zodiacDegree: `19° 5' 47" Aries`,
    },
    saturn: {
      zodiacDegree: `19° 4' 21" Sagittarius`,
    },
    uranus: {
      zodiacDegree: `25° 40' 14" Sagittarius`,
    },
    neptune: {
      zodiacDegree: `7° 31' 22" Capricorn`,
    },
    pluto: {
      zodiacDegree: `7° 55' 40" Scorpio`,
    },
  },
  houses: [
    {
      id: "equal",
      description: "Equal Houses",
      cusps: [
        {
          zodiacDegree: `29° 30' 22" Leo`,
        },
        {
          zodiacDegree: `29° 30' 22" Virgo`,
        },
        {
          zodiacDegree: `29° 30' 22" Libra`,
        },
        {
          zodiacDegree: `29° 30' 22" Scorpio`,
        },
        {
          zodiacDegree: `29° 30' 22" Sagittarius`,
        },
        {
          zodiacDegree: `29° 30' 22" Capricorn`,
        },
        {
          zodiacDegree: `29° 30' 22" Aquarius`,
        },
        {
          zodiacDegree: `29° 30' 22" Pisces`,
        },
        {
          zodiacDegree: `29° 30' 22" Aries`,
        },
        {
          zodiacDegree: `29° 30' 22" Taurus`,
        },
        {
          zodiacDegree: `29° 30' 22" Gemini`,
        },
        {
          zodiacDegree: `29° 30' 22" Cancer`,
        },
      ],
    },
    {
      id: "koch",
      description: "Koch Houses",
      cusps: [
        {
          zodiacDegree: `29° 30' 22" Leo`,
        },
        {
          zodiacDegree: `2º 36' 12" Libra`,
        },
        {
          zodiacDegree: `5º 24' 43" Scorpio`,
        },
        {
          zodiacDegree: `5º 13' 12\" Sagittarius`,
        },
        {
          zodiacDegree: `1º 42' 15" Capricorn`,
        },
        {
          zodiacDegree: `29º 11' 6" Capricorn`,
        },
        {
          zodiacDegree: `29° 30' 22" Aquarius`,
        },
        {
          zodiacDegree: `2º 36' 12" Aries`,
        },
        {
          zodiacDegree: `5º 24' 43" Taurus`,
        },
        {
          zodiacDegree: `5º 13' 12\" Gemini`,
        },
        {
          zodiacDegree: `1º 42' 15" Cancer`,
        },
        {
          zodiacDegree: `29º 11' 6" Cancer`,
        },
      ],
    },
  ],
};
