// astrology-chart-data-generator/src/_tests_/test_examples/ireland.js
export const testExampleIreland = {
  location: "Ireland",
  data: {
    longitude: -6.24889,
    latitude: 53.33306,
    dateTime: {
      year: 1985,
      month: 6,
      day: 8,
      hour: 18,
      minute: 15,
      offset: -1,
    },
  },
  angles: {
    ascendant: {
      zodiacDegree: `10° 46' 6" Scorpio`,
    },
    descendant: {
      zodiacDegree: `10º 46' 6" Taurus`,
    },
    midheaven: {
      zodiacDegree: `27º 19' 41\" Leo`,
    },
    lowerMidheaven: {
      zodiacDegree: `27º 19' 41\" Aquarius`,
    },
  },
  planets: {
    sun: {
      zodiacDegree: `17º 50' 23\" Gemini`,
    },
    moon: {
      zodiacDegree: `29º 11' 44\" Aquarius`,
    },
    mercury: {
      zodiacDegree: `19º 14' 42\" Gemini`,
    },
    venus: {
      zodiacDegree: `2º 9' 6\" Taurus`,
    },
    mars: {
      zodiacDegree: `29º 30' 56\" Gemini`,
    },
    jupiter: {
      zodiacDegree: `16º 56' 27\" Aquarius`,
    },
    saturn: {
      zodiacDegree: `23º 7' 21\" Scorpio`,
    },
    uranus: {
      zodiacDegree: `15º 54' 38\" Sagittarius`,
    },
    neptune: {
      zodiacDegree: `2º 38' 29\" Capricorn`,
    },
    pluto: {
      zodiacDegree: `2º 13' 44\" Scorpio`,
    },
  },
  houses: [
    {
      id: "equal",
      description: "Equal Houses",
      cusps: [
        {
          zodiacDegree: `10º 46' 6" Scorpio`,
        },
        {
          zodiacDegree: `10º 46' 6" Sagittarius`,
        },
        {
          zodiacDegree: `10º 46' 6" Capricorn`,
        },
        {
          zodiacDegree: `10º 46' 6" Aquarius`,
        },
        {
          zodiacDegree: `10º 46' 6" Pisces`,
        },
        {
          zodiacDegree: `10º 46' 6" Aries`,
        },
        {
          zodiacDegree: `10º 46' 6" Taurus`,
        },
        {
          zodiacDegree: `10º 46' 6" Gemini`,
        },
        {
          zodiacDegree: `10º 46' 6" Cancer`,
        },
        {
          zodiacDegree: `10º 46' 6" Leo`,
        },
        {
          zodiacDegree: `10º 46' 6" Virgo`,
        },
        {
          zodiacDegree: `10º 46' 6" Libra`,
        },
      ],
    },
    {
      id: "koch",
      description: "Koch Houses",
      cusps: [
        {
          zodiacDegree: `10º 46' 6" Scorpio`,
        },
        {
          zodiacDegree: `5º 39' 55" Sagittarius`,
        },
        {
          zodiacDegree: `5º 5' 42" Capricorn`,
        },
        {
          zodiacDegree: `27º 19' 41" Aquarius`,
        },
        {
          zodiacDegree: `21º 47' 24" Pisces`,
        },
        {
          zodiacDegree: `16º 22' 17" Aries`,
        },
        {
          zodiacDegree: `10º 46' 6" Taurus`,
        },
        {
          zodiacDegree: `5º 39' 55" Gemini`,
        },
        {
          zodiacDegree: `5º 5' 42" Cancer`,
        },
        {
          zodiacDegree: `27º 19' 41" Leo`,
        },
        {
          zodiacDegree: `21º 47' 24" Virgo`,
        },
        {
          zodiacDegree: `16º 22' 17" Libra`,
        },
      ],
    },
    {
      id: "regiomontanus",
      description: "Regiomontanus Houses",
      cusps: [
        {
          zodiacDegree: `10º 46' 6" Scorpio`,
        },
        {
          zodiacDegree: `4º 48' 37" Sagittarius`,
        },
        {
          zodiacDegree: `12º 0' 10" Capricorn`,
        },
        {
          zodiacDegree: `27º 19' 41" Aquarius`,
        },
        {
          zodiacDegree: `29º 36' 9" Pisces`,
        },
        {
          zodiacDegree: `21º 20' 49" Aries`,
        },
        {
          zodiacDegree: `10º 46' 6" Taurus`,
        },
        {
          zodiacDegree: `4º 48' 37" Gemini`,
        },
        {
          zodiacDegree: `12º 0' 10" Cancer`,
        },
        {
          zodiacDegree: `27º 19' 41" Leo`,
        },
        {
          zodiacDegree: `29º 36' 9" Virgo`,
        },
        {
          zodiacDegree: `21º 20' 49" Libra`,
        },
      ],
    },
    {
      id: "porphyry",
      description: "Porphyry Houses",
      cusps: [
        {
          zodiacDegree: `10º 46' 6" Scorpio`,
        },
        {
          zodiacDegree: `16º 17' 17" Sagittarius`,
        },
        {
          zodiacDegree: `21º 48' 29" Capricorn`,
        },
        {
          zodiacDegree: `27º 19' 41" Aquarius`,
        },
        {
          zodiacDegree: `21º 48' 29" Pisces`,
        },
        {
          zodiacDegree: `16º 17' 17" Aries`,
        },
        {
          zodiacDegree: `10º 46' 6" Taurus`,
        },
        {
          zodiacDegree: `16º 17' 17" Gemini`,
        },
        {
          zodiacDegree: `21º 48' 29" Cancer`,
        },
        {
          zodiacDegree: `27º 19' 41" Leo`,
        },
        {
          zodiacDegree: `21º 48' 29" Virgo`,
        },
        {
          zodiacDegree: `16º 17' 17" Libra`,
        },
      ],
    },
    {
      id: "topocentric",
      description: "Topocentric Houses",
      cusps: [
        {
          zodiacDegree: `10º 46' 6" Scorpio`,
        },
        {
          zodiacDegree: `10º 0' 30" Sagittarius`,
        },
        {
          zodiacDegree: `17º 28' 58" Capricorn`,
        },
        {
          zodiacDegree: `27º 19' 41" Aquarius`,
        },
        {
          zodiacDegree: `29º 34' 13" Pisces`,
        },
        {
          zodiacDegree: `23º 7' 2" Aries`,
        },
        {
          zodiacDegree: `10º 46' 6" Taurus`,
        },
        {
          zodiacDegree: `10º 0' 30" Gemini`,
        },
        {
          zodiacDegree: `17º 28' 58" Cancer`,
        },
        {
          zodiacDegree: `27º 19' 41" Leo`,
        },
        {
          zodiacDegree: `29º 34' 13" Virgo`,
        },
        {
          zodiacDegree: `23º 7' 2" Libra`,
        },
      ],
    },
    {
      id: "campanus",
      description: "Campanus Houses",
      cusps: [
        {
          zodiacDegree: `10º 46' 6" Scorpio`,
        },
        {
          zodiacDegree: `20º 8' 10" Sagittarius`,
        },
        {
          zodiacDegree: `29º 0' 59" Capricorn`,
        },
        {
          zodiacDegree: `27º 19' 41" Aquarius`,
        },
        {
          zodiacDegree: `19º 31' 25" Pisces`,
        },
        {
          zodiacDegree: `11º 53' 43" Aries`,
        },
        {
          zodiacDegree: `10º 46' 6" Taurus`,
        },
        {
          zodiacDegree: `20º 8' 10" Gemini`,
        },
        {
          zodiacDegree: `29º 0' 59" Cancer`,
        },
        {
          zodiacDegree: `27º 19' 41" Leo`,
        },
        {
          zodiacDegree: `19º 31' 25" Virgo`,
        },
        {
          zodiacDegree: `11º 53' 43" Libra`,
        },
      ],
    },
  ],
};
